docker/madoc-omeka-s/usr/local/bin/madoc-installer

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+set -e
+
+## Set up in the Omeka directory.
+cd /srv/omeka;
+
+## PHP Composer
+patch-composer
+composer update --lock --optimize-autoloader --no-dev --prefer-source --no-interaction
+
+## Clear caches.
+composer clearcache
+rm -Rf /srv/omeka/.local /srv/omeka/.config /srv/omeka/.composer
+
+## Ensure files exists.
+mkdir -p /srv/omeka/files
+chmod 777 /srv/omeka/files
```

Comment:
We should really be getting these permissions correct.

Reply:
Any recommendations for the permissions, this script will be run as `www-data` so should own the directory at this point, no idea if Omeka uses the FS ever to list files or how it creates new files. Maybe `775` or `755` ?

Reply:
`770` makes sense, I think. Users that aren't `www-data` have no business looking at these files.